### PR TITLE
INF-729: Add X-Country Header to requests

### PR DESCRIPTION
**Pull request by nickvdyck**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,9 +26,9 @@ jobs:
       - run:
           name: Test
           command: |
-            wget -O /tmp/geolite.tar.gz http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz
-            mkdir -p /var/opt/geolite/ && tar zxvf /tmp/geolite.tar.gz -C /var/opt/geolite/
-            ln -nsf /var/opt/geolite/Geo* /var/opt/geolite/latest
+            mkdir -p /var/opt/geolite/latest
```
Comment (nickvdyck) [inline .circleci/config.yml:29]: The original URL isn't publicly accessible anymore. You now need an account in order to initiate the download. I added the DB into source control, which is for from ideal given the file is almost 60megs. Any ideas?

File: kong/plugins/geoip-mmdb/handler.lua [hunk 2]
```diff
@@ -73,6 +73,12 @@ function plugin:access(conf)
 
   local geo_data = geodb:search_ipv4(remote_addr)
 
+  if geo_data ~= nil and geo_data.country ~= nil and geo_data.country.iso_code ~= nil then
+    kong.service.request.set_header("X-Geo-Country", geo_data.country.iso_code)
+  else
+    kong.service.request.set_header("X-Geo-Country", "ZZ")
```
Comment (nickvdyck) [inline kong/plugins/geoip-mmdb/handler.lua:79]: ISO 3166-1 allows for a range of codes to be user-assigned. I picked `ZZ` here (which I have seen used before) to indicate an invalid or unknown country.

File: .circleci/config.yml (hunk 1 above)
Comment (ludov04) [reply .circleci/config.yml:29]: No this makes sense, I think I ran into the same issue awhile back when updating kong and one step of the dockerfile was to download the geolite database.

I reached the same conclusion and checked out the database into the repo.
https://github.com/datacamp-engineering/kong-gw/commit/7fcbee0b1f64340b6c783165d73e73820e2f2a47#diff-3254677a7917c6c01f55212f86c57fbf

Comment (nickvdyck) [reply .circleci/config.yml:29]: Cool, it seems that you added the gzip file. Which I think is a better idea, so I'm quickly going to change that.

**Comment by nickvdyck** [comment]
> Seems like the ssh key from circleci was gone hence the tests failing.
> I unfollowed, refollowed the project so that circleci would configure it again and reran the build, it's green now!

Awesome thanks, I couldn't figure out why the ssh was suddenly gone and also didn't know how to fix this. This is good to know!

**Pull request merged**